src/hotspot/share/nmt/memTracker.cpp

```diff
@@ -146,12 +146,17 @@ void MemTracker::report(bool summary_only, outputStream* output, size_t scale) {
 void MemTracker::tuning_statistics(outputStream* out) {
   // NMT statistics
   out->print_cr("Native Memory Tracking Statistics:");
-  out->print_cr("State: %s", NMTUtil::tracking_level_to_string(_tracking_level));
-  out->print_cr("Malloc allocation site table size: %d", MallocSiteTable::hash_buckets());
-  out->print_cr("             Tracking stack depth: %d", NMT_TrackingStackDepth);
-  out->cr();
-  MallocSiteTable::print_tuning_statistics(out);
-  out->cr();
+  out->print_cr("State: %s",
+                NMTUtil::tracking_level_to_string(_tracking_level));
+  if (_tracking_level == NMT_detail) {
+    out->print_cr("Malloc allocation site table size: %d",
+                  MallocSiteTable::hash_buckets());
+    out->print_cr("             Tracking stack depth: %d",
+                  NMT_TrackingStackDepth);
+    out->cr();
+    MallocSiteTable::print_tuning_statistics(out);
+    out->cr();
+  }
   out->print_cr("Preinit state:");
```

Comment:
Is there any functional difference here or is it only changing indentation?

Reply:
Hi @jdksjolen, are you referring to the indentation printing the tracking level? I can definitely revert the indentation there.

Reply:
Aha, I see, you added an `if` branch there! It was hard to see with the styling change on top. I think it's fine as is.

Reply:
No worries, thanks for the review!